Library/Formula/mosquitto.rb

```diff
@@ -35,11 +36,6 @@ def caveats; <<-EOD.undent
     mosquitto has been installed with a default configuration file.
     You can make changes to the configuration by editing:
         #{etc}/mosquitto/mosquitto.conf
-
-    Python client bindings can be installed from the Python Package Index:
-        pip install mosquitto
-
-    Javascript client has been removed, see Eclipse Paho for an alternative.
```

Comment:
The Python client is no longer part of mosquitto and the Javascript client is long gone.